### ISSUE TITLE
Orb doesn't work in Alpine images

### ISSUE THREAD

**Issue by jhoos** [codecov/codecov-circleci-orb#8]
The orb doesn't work if it's run inside an Alpine Linux Docker image. CircleCI is running the script in the orb's `run` step using `/bin/sh` (which in Alpine is the "traditional" version of sh, not bash), and the script is using bash-specific syntax and failing:

```
====>> Upload Coverage Results
  #!/bin/sh -eo pipefail
bash <(curl -s https://codecov.io/bash) \
  -f coverages \
  -n ${CIRCLE_BUILD_NUM} \
  -t ${CODECOV_TOKEN} \
  -y .codecov.yml \
  -F  \
  

/bin/sh: syntax error: unexpected "("
Error: Exited with code 2
Step failed
Error: runner failed
```

**Comment by jhoos** [comment]
I should also note that while I have installed bash into my Docker image (via `apk add --no-cache bash`), that doesn't replace `/bin/sh` with `/bin/bash`; they're independent of each other.  So the bash code in the CodeCov script downloaded from the web site would work if the `<(curl ...)` construct can be changed into something `/bin/sh` can handle.

**Comment by teohhanhui** [comment]
Will be fixed by #17
